frontend/resources/translations/en.json

```diff
@@ -657,5 +657,6 @@
       "nature": "Nature",
       "frequentlyUsed": "Frequently Used"
     }
-  }
+  },
+  "grid.settings.Properties": "Properties"
```

Comment:
Thank you for the contribution! I think this token already exists in `en.json` L314. Could you double check it?

Reply:
Due to the limitations of `easy_translation`, we are unable to define the key with dot syntax(.) in the `en.json` file. 

Please remove it or redefine it as 

```
grid {
  settings: {
    Properties: ...
  }
}
```

Reply:
I think I messed that up in inlang. I have removed the changes in L660.